src/jdbc/cubrid/jdbc/driver/CUBRIDClob.java

```diff
@@ -356,13 +361,36 @@ private int readClobPartially(long pos, int length) throws SQLException {
 		return length;
 	}
 
+	private int lobRead(long offset, byte[] buf, int start, int len) throws SQLException {
+		int read_len;
+		long remaining_size;
+
+		remaining_size = lobHandle.getLobSize() - offset;
+
+		if (remaining_size < 0) {
+			return 0;
+		}
+
+		read_len = Math.min((int) remaining_size, len);
+
+		System.arraycopy(lobHandle.getPackedLobHandle(), (int) offset, buf, start, read_len);
+
```

Comment:
don't need to handle the exception for read_len = 0, NullPointException, or IndexOutofBoundsException ?

Reply:
Since this method is private and there is only one caller, I don't think it is necessary to handle exceptions. and I don't think it is likely to be used in any processing other than this issue. The size of read_len you said is always greater than or equal to 0.